azext_iot/_params.py

```diff
@@ -387,17 +387,23 @@ def load_arguments(self, _):
         context.argument(
             "force",
             options_list=["--force", "-f"],
-            help="Overwrites the device's parent device.",
+            help="Overwrites the device's parent device. "
+            "This command parameter has been deprecated and will be removed "
+            "in a future release. Use 'az iot hub device-identity parent set' instead.",
```

Comment:
Arguments also have the option including a deprecate_info object.

For example:

```python
with self.argument_context('test show-parameters') as c:
  c.argument('junk_flag', help='Something we no longer want to support.' deprecate_info=c.deprecate(expiration='2.1.0'))
```

```python
with self.argument_context('test show-parameters') as c:
  c.argument('resource', options_list=['--resource', c.deprecate(target='--resource-id', redirect='--target')])
```